ChangeLog.md

```
@@ -5,6 +5,19 @@ ChangeLog
 
 --------------------------------------------------------------------------------
 
+## v0.2.15 - 2025-12-19
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)